### DISCUSSION TITLE
Download App Privacy information

### DISCUSSION THREAD

**Discussion by sharkusmarkus** [fastlane/fastlane#17748] [General] [answered]
Hello,
I see there is a rather handy new "upload_app_privacy_details_to_app_store" action, which is fantastic, thank you!

I'm wondering if there is / will be the ability to download the privacy data from the app store?

Why would someone want this? It's simply if someone has used App Store Connect to enter the information, then it would be great to be able to download and store that, much in the same way you may already do that for the other app metadata

**Comment by dinsen** [comment, marked as answer]
Looks like someone is already working on it https://github.com/fastlane/fastlane/pull/17746

**Reply by sharkusmarkus** [reply]
Thank you for the heads-up on that!
